corehq/ex-submodules/dimagi/utils/rate_limit.py

```diff
@@ -1,7 +1,7 @@
-from datetime import datetime, timedelta
 from dimagi.utils.couch.cache.cache_core import get_redis_client
 
 
+# todo: should this be replaced project_limits.RateLimiter?
```

Comment:
not sure, this function is a much simpler limiter. It looks like it's only used by the SMS task.

Reply:
yeah i saw that which made me think it'd be easy to just port that one usage over to the other one. my instinct was that we'd benefit from economies of scale if there was only one way to do rate limiting (more consistent, tested, standardized, documented, etc.).

I was imagining that you could maybe create a function with the same API that used the other thing under the hood. so you get the benefits of simplicity + consistency.